patches/chrome-browser-notifications-notification_platform_bridge_mac.mm.patch

```diff
@@ -2,7 +2,7 @@ diff --git a/chrome/browser/notifications/notification_platform_bridge_mac.mm b/
 index 08078c5f168ebf562fd19c0ebd621f1f7a5a6b5d..8bcd16b11888402fa48926564e13f78657442a05 100644
 --- a/chrome/browser/notifications/notification_platform_bridge_mac.mm
 +++ b/chrome/browser/notifications/notification_platform_bridge_mac.mm
-@@ -226,6 +226,7 @@ void NotificationPlatformBridgeMac::Display(
+@@ -226,6 +226,7 @@ @interface AlertDispatcherImpl : NSObject<AlertDispatcher>
```

Comment:
?

Reply:
It was in a different commit which is because the previous Chrome rebase did not rebase these files properly.